src/components/atoms/AtomDoughnutChart.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <canvas ref="ctx" width="400" height="400"></canvas>
+</template>
+<script lang="ts" setup>
+import OrganismStatistic from '../organisms/OrganismStatistic.vue';
+import Chart from 'chart.js/auto';
+import { LayoutPosition } from 'chart.js';
+import { onMounted, PropType, ref } from 'vue';
```

Comment:
Change order of imports
```suggestion
import { onMounted, PropType, ref } from 'vue';
import Chart from 'chart.js/auto';
import { LayoutPosition } from 'chart.js';
import OrganismStatistic from '../organisms/OrganismStatistic.vue';
```